optimum/onnxruntime/modeling_ort.py

```diff
@@ -213,8 +276,9 @@ def _from_pretrained(
             )
             kwargs["model_save_dir"] = Path(model_cache_path).parent
             kwargs["latest_model_name"] = Path(model_cache_path).name
-            model = ORTModel.load_model(model_cache_path)
+            model = ORTModel.load_model(model_cache_path, **kwargs)
```

Comment:
I think here we should define `self.provider` as well. That way a user can already access `model.provider` otherwise it wouldn't make much sense to add it in the `to` method.

Reply:
I define it in the `__init__` from `ORTModel`: https://github.com/fxmarty/optimum/blob/21f4d919b202561968258fb6e8dbce67130e957b/optimum/onnxruntime/modeling_ort.py#L98 . Does it sound good?